### ISSUE TITLE
scisharp.tensorflow.redist-linux-gpu 2.5 inludes windows binary instead of linux binaries

### ISSUE THREAD

**Issue by rhythmnewt** [SciSharp/TensorFlow.NET#813]
Shouldn't these be a bunch of .so files like in 2.3.1? Edit: changed to 2.3.1 since there was no 2.4.1 for GPU Linux (why?)

![image](https://user-images.githubusercontent.com/1319322/119899201-d04cad00-bf10-11eb-8b42-c9a00e8f87bf.png)

Thanks

**Comment by rhythmnewt** [comment]
Getting correct binaries fixed my runtime issues:

Dockerfile steps for anyone struggling with the same thing. Insert after copying your .net bin/ to /app
```
WORKDIR /tensorflowlib
RUN curl -fSsl -O https://storage.googleapis.com/tensorflow/libtensorflow/libtensorflow-cpu-linux-x86_64-2.5.0.tar.gz
RUN tar xvzf libtensorflow-cpu-linux-x86_64-2.5.0.tar.gz
RUN cp ./lib/*.* /app/runtimes/linux-x64/native
```

**Comment by rhythmnewt** [comment]
Seriously? It's been FOUR MONTHS since this was reported and no updated nuget package with fixed binaries???? SMH

**Comment by Oceania2018** [comment]
@rhythmnewt We can release the Linux gpu package due to binary file size limitation by NuGet. Please raise issue to NuGet.

**Comment by rhythmnewt** [comment]
I am closing this issue as resolved, since 2.11 solved the nuget binary size issue by splitting the gpu package into multiple dependencies about 10 months ago and there's a workaround for 2.5 without going through nuget.org.

https://www.nuget.org/packages/SciSharp.TensorFlow.Redist-Linux-GPU/2.11.1
https://github.com/SciSharp/TensorFlow.NET/issues/966
